pkg/catalog/model.go

```diff
@@ -56,6 +58,38 @@ type Tag struct {
 	CommitID string
 }
 
+// AddressType is the type of an entry address
+type AddressType int32
+
+const (
+	// AddressTypeUnknown indicates that the address might be relative or full.
+	// Used only for backward compatibility and should not be used for creating entries.
+	AddressTypeUnknown AddressType = 0
+
+	// AddressTypeRelative indicates that the address is relative to the storage namespace.
+	// For example: "/foo/bar"
+	AddressTypeRelative AddressType = 1
+
+	// AddressTypeFull indicates that the address is the full address of the object in the object store.
+	// For example: "s3://bucket/foo/bar"
+	AddressTypeFull AddressType = 2
+)
+
+func (at AddressType) IsRelative() *bool {
```

Comment:
The name `Is*` is confusing for a function returning a tristate.

Reply:
Done.